cellprofiler/modules/trackobjects.py

```diff
@@ -4,7 +4,9 @@
 TM_DISTANCE = 'Distance'
 TM_MEASUREMENTS = 'Measurements'
 TM_LAP = "LAP"
-TM_ALL = [TM_OVERLAP, TM_DISTANCE, TM_MEASUREMENTS, TM_LAP]
+TM_FOLLOWNEIGHBORS = "Follow Neighbors"
+TM_ALL = [TM_OVERLAP, TM_DISTANCE, TM_MEASUREMENTS, TM_LAP, TM_FOLLOWNEIGHBORS]
+DOC_YEASTTOOLBOX_WEB = "www.cellprofiler.org/yeasttoolbox/"
```

Comment:
This URL does not exist. You'll need to create content for it here: https://github.com/CellProfiler/cellprofiler.github.com